README.adoc

```diff
@@ -184,33 +206,39 @@ include::finish/src/main/java/io/openliberty/guides/jpaguide/dao/EventDao.java[t
 ----
 
 In order to use the entity manager at runtime, it must be injected into our EJB component through the
-`@PersistenceContext` annotation. Since the entity manager is container-managed, the EJB container
-will ensure that the methods are executed within an active transaction which is why it is not necessary
-to explicitly begin, commit, or rollback transactions. At the end of an EJB operation, the persistence 
-context will be notified that the transaction is committing and the persistence provider, EclipseLink,
-will flush the changes to the database.
+`@PersistenceContext` annotation. Since the stateless session bean is configured to manage the global
+transactions, the EJB container will ensure that the required methods are executed within the boundaries
+of an active global transaction, which is why it is not necessary to explicitly begin, commit or rollback
+transactions. At the end of a session bean method invocation, the persistence context will be notified
+that the transaction is committing, and will flush any changes made to entity instances it is managing
+to the database.
 
 The `EventDao.java` class has a method for each CRUD operation, so let's break them down:
 
 * The `createEvent()` method persists an instance of the `Event` entity class to the datastore by
-calling the `persist()` method on an `EntityManager` instance. The argument passed into the `persist()`
-method must be an instance of an entity class, otherwise an `IllegalArgumentException` will be thrown.  
+calling the `persist()` method on an EntityManager instance. The event objects state will be 
+changed from new to managed and changes to it are now tracked by the EntityManager. 
 
 * The `readEvent()` method will return an instance of the `Event` entity class with the specified primary 
 key by calling the `find()` method on an `EntityManager` instance. The `find()` method accepts both the entity 
-class and the primary key as arguments. If the event instance is not found, `Null` will be returned. Similarily,
-the `readAllEvents()` method returns a list of instances of the `Event` entity class using a query
-specified in the `@NamedQuery` annotation on the `Event` class. 
+class and the primary key as arguments. If the state of the entity being searched for is managed, the event 
```

Comment:
Depends.  A simple em.find() (which is a single-entity single-identity fetch request) will certainly check the persistence context first before issuing SQL statements to the database, but a query will still issue SQL statements to the database.